models/VolunteeringRequest.js

```diff
@@ -0,0 +1,95 @@
+
+/*!
+ * Module dependencies
+ */
+var mongoose = require('mongoose');
+var Schema = mongoose.Schema;
+
+/**
+ * VolunteeringRequest schema
+ */
+var VolunteeringRequestSchema = new Schema({
+  location: {
```

Comment:
Shouldn't we use address here? (as a string)

Reply:
If we will do it with coordinates then every time the user will enter an address or request one we'll have to translate it which is waste of time.
If we will do it with address then every time we will just represent that field and ONLY when it comes to the map, we will translate it.

tell me what you think